lib/shopify-cli/tasks/select_org_and_shop.rb

```diff
@@ -8,6 +8,9 @@ class SelectOrgAndShop < ShopifyCli::Task
       def call(ctx, organization_id: nil, shop_domain: nil)
         @ctx = ctx
         return response(organization_id.to_i, shop_domain) unless organization_id.nil? || shop_domain.nil?
+        if Shopifolk.check && wants_to_run_against_shopify_org?
```

Comment:
This impacts the following commands:

* Node - Forms - Create
* Rails - Forms - Create
* Populate Resource